### DISCUSSION TITLE
Geting the World Map with get_eurostat_geospatial()

### DISCUSSION THREAD

**Discussion by guaribr** [rOpenGov/eurostat#255] [Ideas]
Would be a good idea to also get the world map using the `get_eurostat_geospatial()` function.
Thanks

**Comment by pitkant** [comment]
Hi, thank you for your suggestion! I can see where you're coming from. The way I see it `get_eurostat_geospatial` is meant to be a helpful function for easy visualisation of data downloaded from Eurostat - meaning that it mainly concerns only Europe instead of the whole world. (Without any boundary box settings the default map output that includes French Guyana and other far-away territories is, at least in my opinion, from this point-of-view a bit problematic and something should be done about that)

Maybe function author @muuankarski can comment on this?

Perhaps you would find the giscoR package useful? See this introduction: [Get started with giscoR](https://ropengov.github.io/giscoR/articles/giscoR.html)

**Reply by guaribr** [reply]
Thanks @pitkant! I'll will explore giscoR. Sometimes is useful to also show EU neighboring countries and is a pitty that we can not plot them all at once using get_eurostat_geospatial(). The other cool feature is that we can quickly get the data into a df and share it. Once again, thanks @pitkant and @muuankarski.

**Comment by antagomir** [comment]
I agree with @pitkant about the scope of the eurostat functions and thanks for the giscoR tip. 

Having opportunities to map broader regions would certainly be useful but imo. better addressed in the context of a separate package (and some solutions exist already).
